### DISCUSSION TITLE
Personalizador online para Tachyons

### DISCUSSION THREAD

**Discussion by luizbills** [frontendbr/forum#944] [Perguntas e Respostas]
Alguém aqui usa [tachyons](http://tachyons.io)? Eu comecei a analiza-lo e acabei gostando muito. Para quem não conhece, ele é um framework de **functional css**, alguns também chamam de **utility-first css** ou **atomic css** (não confundir com [**atomic design**](http://bradfrost.com/blog/post/atomic-web-design/)). Recomendo este [artigo](https://frontstuff.io/in-defense-of-utility-first-css) para introdução.

Enfim... o _tachyons_ (assim como outros frameworks) são bem opinativos, então resolvi criar um [personalizador](https://tachyons.luizpb.com/) do Tachyons. Primeiro criei um [plugin para WordPress](https://github.com/luizbills/wp-modular-css) (meu ganha pão), mas resolvi criar uma *frontend GUI* (usando o plugin como backend) para que mais pessoas tivessem acesso, sem ter que usar/saber wordpress.

---

_Curiosidade: o [novo bootstrap](https://getbootstrap.com/docs/4.0/utilities/borders/) está usando algumas utility classes (similiar ao que o tachyons faz). 
PS: Acredito que outros frameworks também devam ter algumas "utilitárias", porém citei o bootstrap só por causa da popularidade dele._

**Comment by felipefialho** [comment]
Linkando: #945
